### PR TITLE
Experiment Results Detail Page

### PR DESCRIPTION
**Pull request by neilkakkar**
## Changes

Still need to clean this up a bit, but wanted to put up a WIP - so @liyiy can see as well.

Opted to return filters from the API as I couldn't find a decent way of getting these otherwise

image

Need to figure out a way of removing that date range box^. Since the funnel results are cached, no changes can be made to this insight (which is what we want)

## How did you test this code?

YOLO

### CODE REVIEW

File: frontend/src/scenes/experiments/Experiment.tsx [hunk 1]
```diff
@@ -131,6 +133,24 @@ export function Experiment(): JSX.Element {
                 <div>Owner: {experimentData.created_by?.first_name}</div>
                 <div>Feature flag key: {experimentData?.feature_flag_key}</div>
             </div>
+
+            {experimentResults && (
+                <BindLogic
+                    logic={insightLogic}
+                    props={{
+                        dashboardItemId: undefined,
+                        filters: { ...experimentResults.filters, insight: 'FUNNELS', display: 'FunnelViz' },
```
Comment (neilkakkar) [inline frontend/src/scenes/experiments/Experiment.tsx:142]: filters currently default to returning `ActionsLineGraph` and `TRENDS`. Assuming these are defaults, make sure FE always sends these two params on Experiment creation

File: frontend/src/scenes/experiments/Experiment.tsx [hunk 2]
```diff
@@ -1,40 +1,48 @@
...
 import React, { useRef } from 'react'
+import { InsightContainer } from 'scenes/insights/InsightContainer'
+import { insightLogic } from 'scenes/insights/insightLogic'
+import { SceneExport } from 'scenes/sceneTypes'
 import { userLogic } from 'scenes/userLogic'
 import { PropertyFilter } from '~/types'
 import './Experiment.scss'
 import { experimentLogic } from './experimentLogic'
 
+export const scene: SceneExport = {
```
Comment (liyiy) [inline frontend/src/scenes/experiments/Experiment.tsx:16]: aren't most of the changes in this PR from https://github.com/PostHog/posthog/pull/7584? It's hard to see which code changes are from this PR vs the old one here 😞

Comment (neilkakkar) [reply frontend/src/scenes/experiments/Experiment.tsx:16]: Hmm, whoops, it was based on top of the other one. Give me a moment, let me rebase!

File: frontend/src/scenes/experiments/Experiment.tsx [hunk 3]
```diff
@@ -37,220 +39,232 @@ export function Experiment(): JSX.Element {
     )
     const { isStepsEmpty, filterSteps, filters } = useValues(funnelLogic(insightProps))
 
-    return (
+    return experimentId === 'new' ? (
```
Comment (neilkakkar) [inline frontend/src/scenes/experiments/Experiment.tsx:42]: noooo, chose the wrong branch, ugh xP

**Comment by liyiy** [comment]
Obviously still some design improvements to make but date range filter is removed and so is the details table for this view, let's try to get this in to avoid more merge conflicts in the logic file?

Screen Shot 2021-12-13 at 10 24 08 PM

**Review by liyiy — APPROVED** [review]

File: frontend/src/scenes/insights/InsightTabs/InsightDisplayConfig.tsx [hunk 4]
```diff
@@ -94,7 +96,7 @@ export function InsightDisplayConfig({
     return (
         <div className="display-config-inner">
             <div className="display-config-inner-row">
-                {showDateFilter[activeView] && (
+                {showDateFilter[activeView] && !disableTable && (
```
Comment (neilkakkar) [inline frontend/src/scenes/insights/InsightTabs/InsightDisplayConfig.tsx:99]: I think it's kinda weird that disableTable disables the date range thingy, but I guess it's fine for now, will come and clean this up once the datefilters are fixed!

**Pull request merged**